pkg/terraform/terraform.go

```diff
@@ -126,8 +126,9 @@ func unpackAndInit(dir string, platform string) (err error) {
 
 	os.Setenv("TF_CLI_CONFIG_FILE", filepath.Join(dir, "terraform.rc"))
 
-	// XXX: This is only here for debugging CI
+	// XXX: These are only here for debugging CI
 	os.Setenv("TF_LOG", "trace")
+	os.Setenv("TERRAFORM_LOCK_FILE_PATH", dir)
```

Comment:
Rather than setting this env variable, can we change directories into the temp directory where all of the terraform files are and then change back to the previous working directory when terraform is done? That would keep us from having to carry the patch supporting this environment variable. It would also mean that we don't need to set the `OverrideDataDir` field in the terraform command meta.

Reply:
Here are the changes that I made to test changing the directory, if you are interested in doing it this way.
https://github.com/staebler/installer/commit/5917fc333385c82612e51cbc83908dc7001fe614